sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -580,11 +603,14 @@ public CosmosAsyncScripts getScripts() {
      */
     public CosmosPagedFlux<CosmosConflictProperties> readAllConflicts(CosmosQueryRequestOptions options) {
         return UtilBridgeInternal.createCosmosPagedFlux(pagedFluxOptions -> {
+            pagedFluxOptions.setTracerInformation(this.getDatabase().getClient().getTracerProvider(),
```

Comment:
@simplynaveen20 Was thinking about this a little bit. Would it be acceptable for Cosmos behavior to throw an NPE if the tracer failed to load?

As tracing being an added functionality would it rather be acceptable to log that the tracer could not be loaded rather than to throw and make it completely impossible for the users to use this functionality just because we want to mandate a code pattern for the developers?

Reply:
Its not about tracer , its about tracerProvider. Null pointer wont come for non tracer user. It will only come for dev who will not use the tracer pipeline, irrespective of tracer present or not

Reply:
Null pointer exceptions should be an eager check.
cc: @kushagraThapar

Reply:
Agreed, we should check them upfront. 
Updated the tracking issue to include this as well - https://github.com/Azure/azure-sdk-for-java/issues/13031